app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -33,6 +33,66 @@ def fetch(hash)
     end
   end
 
+  def store_ids_for_new_dto_records(records, dto_collection)
```

Comment:
hm, not sure, lets do that as next step probably, all the changed methods has dto in it now